terraform-modules/lambda-accounts/code/accounts-ips/accounts-ips.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+import os
+import json
+import boto3
+
+from utils.utils_aws import list_accounts
+
+state_machine_arn = os.environ["STATE_MACHINE_ARN"]
+
+
+def lambda_handler(event, context):  # pylint:disable=unused-argument
+    # lists all AWS accounts in organization and sends in JSON format to Step Functions state machine
+
+    accounts = list_accounts()
+    print(accounts)
+
+    input_data = []
+
+    keys_to_remove = ["Arn", "Email", "Status", "JoinedMethod", "JoinedTimestamp"]
+
+    for account in accounts:
+        for key in keys_to_remove:
+            del account[key]
```

Comment:
Something to consider here is that if any keys get added to the `accounts` list then they will also be send downstream.  Would a whitelist rather than a blacklist be safer here (i.e. only copy the keys we know we want into a new object and append that to the `input_data` list)?

Reply:
updated